### ISSUE TITLE
file_get_contents_curl fails

### ISSUE THREAD

**Issue by michael-cannon** [michael-cannon/typo3-importer#3]
https://aihr.us/wordpress/typo3-importer-for-wordpress-2-0-1-update/#comment-138639

**Comment by subharanjanm** [comment]
Not sure if this is related to the same issue, but maybe we need to check if cURL is enabled or not. 
https://wordpress.org/support/topic/check-if-curl-is-installed?replies=2

**Comment by michael-cannon** [comment]
Yup, this is a back and forth PHP configuration issue. None of the top or
preferred methods are consistently available and we should look into more
fallbacks.
